cartography/driftdetect/add_shortcut.py

```diff
@@ -49,7 +45,11 @@ def add_shortcut(storage, shortcut_serializer, query_directory, alias, filename)
     shortcut_path = os.path.join(query_directory, "shortcut.json")
     shortcut_data = storage.load(shortcut_path)
     shortcut = shortcut_serializer.load(shortcut_data)
-    shortcut.shortcuts[alias] = filename
+    fp = shortcut.shortcuts.get(filename, filename)
```

Comment:
How about this case?
Suppose:
```
filename = 'a'
alias = 'a'
```

and `shortcut.shortcuts` is already defined:
```
shortcut.shortcuts = {
 'a': 'somethingelse.json'
}
```

Then,
```
fp = shortcut.shortcuts.get('a', 'a') ==> 'somethingelse.json'
```

so `somethingelse.json` is still the filename and not `a`, so the code did not update the shortcut.

Reply:
Yeah I just noticed an issue where if the shortcut is the name of another file that produces that error. I added a check for it, but I'm worried that there might be too many IO calls between the storage unit and the program. It's a pretty pathological counter example which I don't think would occur in practice, and I'm debating between the check and just adding a warning to the help documentation for the subparser. What do you think?